src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -72,6 +75,12 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/databases/*/admin/record", "GET", AuthorizationStatus.DatabaseAdmin)]
+        public async Task GetSettingForClient()
+        {
+            await SendDatabaseRecord(Database.Name, ServerStore, HttpContext, ResponseBodyStream());
```

Comment:
Sanity check:
@ayende The `/admin/databases` endpoint is only accessible by operator. Now we are exposing to DatabaseAdmin record of the database that they can manage. Which sounds reasonable, do you have any objections? Is there anything in the record that we would not want to expose to DatabaseAdmin? I cannot think of anything, but would like an additional person to confirm.

Reply:
DatabaseAdmin should be able to see everything there, yes.

Reply:
but what is the scenario where you have only db admin right and need it?

Reply:
We are using this to download settings, but I didn't want to have X-number of endpoints for all of the properties, so we decided to expose whole record and just take the value of the property in the client
e.g. in this case admin can get settings and then change them

Reply:
Okay, make sense.